client/src/pages/Map/MapLayout.js

```diff
@@ -105,7 +105,9 @@ export default function WrappedMapLayout() {
     <ErrorBoundary>
       <NewTreeProvider>
         <UserLocationProvider>
-          <MapLayout />
+          <MapboxManagerProvider>
```

Comment:
I think the latest best practice recommendation is to not use redux except for very state-heavy front-end applications. I've read a bunch of blogs about zustand or other lightweight stores being a better alternative for apps like ours. 

That said, not sure what the tipping point would be. I think we can get away with contexts for now, but that's also because I don't want to own incorporating stores into the codebase right at this minute.

We currently have NewTree, UserLocation, and MapboxManager as contexts. Let's say we seriously consider moving to some state library after we add a couple more?